spec/Powertrain/Battery.vspec

```diff
@@ -70,12 +83,30 @@ CellVoltage.Min:
   unit: V
   description: Current voltage of the battery cell with lowest voltage.
 
+CellVoltage.IdMin:
+  datatype: uint16
+  type: sensor
+  description: Identifier of the battery cell with lowest voltage.
+  comment: Identifier is supposed to be relative index of the cell, starting with 0 the first cell.
+
 CellVoltage.Max:
   datatype: float
   type: sensor
   unit: V
   description: Current voltage of the battery cell with highest voltage.
 
+CellVoltage.IdMax:
+  datatype: uint16
+  type: sensor
+  description: Identifier of the battery cell with highest voltage.
+  comment: Identifier is supposed to be relative index of the cell, starting with 0 the first cell.
+
+CellVoltage.CellVoltage:
```

Comment:
This seems to cause a problem for ddsidl - we get a struct CellVoltage within a module CellVoltage which causes problem for idlc.

To me this seems like a bug in ddsidl - the VSS is correct, ddsidl needs to handle it. Off course we could avoid the problem by selecting a different name, but that is not the long term solution as I see it.